sdk/core/core-http/lib/util/atomHandler.ts

```diff
@@ -0,0 +1,63 @@
+import { Constants } from "./constants";
+
+/**
+ * Utility to deserialize the given JSON content even further based on
+ * if it's a single `entry` or `feed`
+ * @param {object} xmlInJson
+ * */
+export function parseResultFromAtomResponse(atomResponseInJson: any): any {
+  if (!atomResponseInJson) {
+    return;
+  }
+
+  if (atomResponseInJson.feed) {
+    return parseFeedResult(atomResponseInJson.feed);
+  }
+
+  if (atomResponseInJson.entry) {
+    return parseEntryResult(atomResponseInJson.entry);
+  }
+
+  throw new Error("Unrecognized result: " + atomResponseInJson);
+}
+
+function parseEntryResult(entry: any): any {
+  var contentElementName = Object.keys(entry.content).filter(function(key) {
+    return key !== Constants.XML_METADATA_MARKER;
+  })[0];
```

Comment:
About this, the entry.content item we are dealing with is the parsed JSON representation of the XML.
For any given object, looks like there would be just one key value pair represented using markers $ and _ respectively. 
Filter likely returns an array and looking at usage, since this is used only for parsing a single entry and not array, it seems safe to try and access the first element when present or return nothing otherwise.